### ISSUE TITLE
IOS 18.1 Beta 5

### ISSUE THREAD

**Issue by defnotelonmusk** [straight-tamago/misakaX#271]
I assume IOS 18.1 Beta 5 will be updated soon for iPhone models, is it in the works currently?

**Comment by DiegoMA2901** [comment]
> I assume IOS 18.1 Beta 5 will be updated soon for iPhone models, is it in the works currently?

Nope, we're waiting to misaka X to release a version that works with the new beta 5

**Comment by defnotelonmusk** [comment]
How long could that take?
